app/components/dashboard/hypertension/downloads_dropdown_component.html.erb

```diff
@@ -3,41 +3,42 @@
     <i class="fa fa-arrow-circle-down w-16px ta-center mr-4px c-blue"></i>
     Downloads <% if @region.diabetes_management_enabled? %>(HTN)<% end %>
   </a>
+
   <div class="dropdown-menu dropdown-menu-right" aria-labelledby="dropdownMenuLink">
 
     <h6 class="dropdown-header c-grey-dark">
       CSV files
     </h6>
 
-    <%= link_to(reports_region_download_path(@region, report_scope: params[:report_scope], period: :quarter, format: :csv), class: "dropdown-item") do %>
+    <%= link_to(region_download_path, class: "dropdown-item") do %>
       <i class="far fa-file w-16px ta-center mr-8px c-blue"></i>
       Quarterly cohort report
     <% end %>
 
-    <% if @region.district_region? && current_admin.feature_enabled?(:monthly_district_report)%>
-      <%= link_to(reports_region_monthly_district_report_path(@region, report_scope: params[:report_scope], period: params.dig(:period, :value), format: :zip), class: "dropdown-item") do %>
+    <% if region.district_region? && current_admin.feature_enabled?(:monthly_district_report)%>
```

Comment:
Since we are using components, you can move this condition into a methion, maybe something like `show_monthly_district_report?`

This way, we can keep our view clean and easy to read.

Same of other conditionals.